Add org profiles #4370, start to refactor user profiles

stephanegigandet opened the pull request:
This adds profiles for organizations (see #4370 for details and screenshot).

I also started some refactoring to improve user profiles, to make it easier to redesign them and add fields (such as the map: 
#4351 )

teolemon commented:
![image](https://user-images.githubusercontent.com/1689815/96163911-c72b4480-0f1a-11eb-8596-fe073f68d7a2.png)

stephanegigandet commented:
I replaced the table and removed all the responsive columns in the org profile edit form.
That way the behavior is the same at all sizes.
I think we should do the same for the user edit form too.

![image](https://user-images.githubusercontent.com/8158668/96234663-05b41400-0f9a-11eb-878e-97c7677db972.png)
